mathics/__init__.py

```diff
@@ -1,10 +1,17 @@
-# -*- coding: utf8 -*-
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import absolute_import
 
-# force utf8 encoding
 import sys
-import codecs
-writer = codecs.getwriter("utf-8")
-sys.stdout = writer(sys.stdout)
+import platform
+
+# TODO fix this ugly hack
+# import codecs
+# writer = codecs.getwriter("utf-8")
+# sys.stdout = writer(sys.stdout)
```

Comment:
This was an ugly hack to get unicode input working. I'm not sure if there's a better approach.